packages/aquedux-client/src/reducers/index.js

```diff
@@ -1,41 +1,23 @@
 import actionTypes from '../constants/actionTypes'
 
+import configManager from '../managers/configManagers'
+
 export const initialState = {
   definitions: {},
   subscription: [],
   pongTimestamp: Date.now(),
   pingTimestamp: Date.now(),
-  pingState: 'ok',
-  config: {
-    pingAwareActionTypes: {},
-    timeout: 5000,
-    endpoint: undefined
-  }
+  pingState: 'ok'
+}
+const { pingAwareActionTypes, timeout } = configManager.getConfig()
+
+const isPingAwareActionType = actionType => {
+  const internalActionTypes = [actionTypes.AQUEDUX_PING, actionTypes.AQUEDUX_CHANNEL_SNAPSHOT]
+
+  return internalActionTypes.includes(actionType) || pingAwareActionTypes.includes(actionType)
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes
^ It looks like `Array.includes` is not supported on IE.

We should use indexOf instead.

Reply:
Or use an Object and `hasOwnProperty` 😉